feature/experimental/interface/my_station_mac/ate_tests/my_station_mac_test/my_station_mac_test.go

```diff
@@ -196,7 +197,7 @@ func TestMyStationMAC(t *testing.T) {
 	gnmi.Replace(t, dut, gnmi.OC().System().MacAddress().RoutingMac().Config(), myStationMAC)
 
 	t.Logf("Verify configured MyStationMAC through telemetry")
-	if got := gnmi.Get(t, dut, gnmi.OC().System().MacAddress().RoutingMac().State()); got != myStationMAC {
+	if got := gnmi.Get(t, dut, gnmi.OC().System().MacAddress().RoutingMac().State()); strings.ToUpper(got) != myStationMAC {
```

Comment:
Why not just
if !strings.EqualFold(got, myStationMAC) {
 .....
}
so it supports all cases?